Runtime/Scripts/Types/RGBot.cs

```diff
@@ -21,6 +21,17 @@ public override string ToString()
             return "" + id + " - " + name;
         }
     }
-    
-}
 
+    /// <summary>
+    /// Represents a bot record that comes from the Regression Games service.
+    /// </summary>
+    /// <remarks>
+    /// Bots that come from the Regression Games service include some additional metadata that local bots do not have.
+    /// </remarks>
+    [Serializable]
+    public class RGRemoteBot : RGBot
```

Comment:
RGBot is in the `Types` folder/namespace... These `Types` represent the same entities that we use in RGService.  Long story short, we don't need RGRemoteBot vs RGBot as RGBot is already meant to represent those remote records.

Reply:
We do also use it for local bots so I was a little hesitant to just combine them. It's probably not a big deal though. Happy to just merge these types and keep calling them RGBot for now. It may become useful to separate the local and remote metadata for a synced bot but we can do that when we have a case that necessitates it.

Reply:
let's definitely have it as just `RGBot` / `RGBotList`; these types are meant to represent the same record as RGService's version of `Bot` with the same fields/etc whether they are being used for a Local bot or Remote